src/architecture/clients.md

```diff
@@ -68,17 +74,37 @@ implemented in Exonum light client.
   that links the response to the block header.
 3. The client, on receiving the response from the blockchain, *verifies the structure*
   and *validates cryptographic proofs* for the response.
-4. The verification procedure includes *checking whether a returned response
+  The verification procedure includes *checking whether a returned response
   is stale*. This is accomplished by calculating the median of timestamps recorded
   in `Precommit`s and comparing it against the local time of the client.
   If the median time in `Precommit`s is too far in the past, the response
   is considered stale, and its verification fails.
-5. The result of checks is shown in the user interface
+4. The result of checks is shown in the user interface
```

Comment:
I'd suggest to clarify:

> The result of checks and the data retrieved from the full node is shown in the user interface

to more closely match what is displayed in the figure.